### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by jwilk**

### CODE REVIEW

File: content/charsets/contents.lr [hunk 1]
```diff
@@ -9,13 +9,13 @@ body:
 
 ## UTF-8
 
-Modern terminals support UTF-8 as an univeral character set.
+Modern terminals support UTF-8 as a universal character set.
```
Comment (istoph) [inline content/charsets/contents.lr:12]: You are certainly right, but why not an "an", because the next one is a "u"?

**Review by istoph — APPROVED** [review]

File: content/charsets/contents.lr (hunk 1 above)
Comment (jwilk) [reply content/charsets/contents.lr:12]: This "u" looks like a vowel, but it doesn't sound like one, and it's pronunciation that matters.
So it's *a universal charset* but *an upper-case character*; *a house* but *an hour*.

**Comment by textshell** [comment]
Thanks. That's quite a bit. I'll have to look at this when i've got a bit of time.

For some reason the deploy preview comment does not get added automatically at the moment.

Deploy preview: https://deploy-preview-12--terminalguide.netlify.app/

**Review by istoph — APPROVED** [review]

File: content/seq/csi_x5D_square_bracket/contents.lr [hunk 2]
```diff
@@ -10,7 +10,7 @@ body:
 <tr><td><code>cmd</code> = 10</td><td>Set the frequency for {{seq_link('a_c0-g')}} to <code>param</code> Hz. If <code>param</code> is not specified reset to default.</td></tr>
 <tr><td><code>cmd</code> = 11</td><td>Set the duration for {{seq_link('a_c0-g')}} to <code>param</code> ms. If <code>param</code> = 0 or >= 2000 disable bell. If <code>param</code> is not specified reset to default.</td></tr>
 <tr><td><code>cmd</code> = 12</td><td>Switch visible console to console number <code>param</code>.</td></tr>
-<tr><td><code>cmd</code> = 13</td><td>Reset console blanking timer and unblank the console if needed.<br>This affects als virtual terminals.</td></tr>
+<tr><td><code>cmd</code> = 13</td><td>Reset console blanking timer and unblank the console if needed.<br>This affects also virtual terminals.</td></tr>
```
Comment (textshell) [inline content/seq/csi_x5D_square_bracket/contents.lr]: Actually this should be "all"

**Review by textshell — COMMENTED** [review]
Thanks for submitting these corrections.

**Pull request merged**